README.md

```diff
@@ -98,12 +98,20 @@ Within your engine's `addon` directory, add a new `engine.js` file:
 ```js
 import Engine from 'ember-engines/engine';
 import Resolver from 'ember-engines/resolver';
+import loadInitializers from 'ember-load-initializers';
 
-export default Engine.extend({
-  modulePrefix: 'ember-blog',
+let Eng;
```

Comment:
Can you tweak this to something like the following:

```js
const modulePrefix = 'ember-blog';
const Eng = Engine.extend({
  modulePrefix,
  Resolver
});

loadInitializers(Eng, modulePrefix);
```

Reading it top to bottom like this seems easier to my brain 😝